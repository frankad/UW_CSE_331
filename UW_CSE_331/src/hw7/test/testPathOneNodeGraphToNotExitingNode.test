# Test a case to find path using oneNodeGraph from the node that exist inside
# the graph to some not existing nodes say, Jeff and David

# Load oneNodeGraph.tsv
LoadGraph oneNodeGraph oneNodeGraph.tsv

# Find the minimum-cost path from  existing(Eva) to non-existing(Jeff or David) node and vice versa
FindPath oneNodeGraph Eva Jeff
FindPath oneNodeGraph David Eva 