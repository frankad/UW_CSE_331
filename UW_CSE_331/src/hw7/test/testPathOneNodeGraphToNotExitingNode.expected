# Test a case to find path using oneNodeGraph from the node that exist inside
# the graph to some not existing nodes say, Jeff and David

# Load oneNodeGraph.tsv
loaded graph oneNodeGraph

# Find the minimum-cost path from  existing(Eva) to non-existing(Jeff or David) node and vice versa
unknown character Jeff
unknown character David